src/pages/Cart/components/BottomSheet.tsx

```diff
@@ -58,12 +60,18 @@ export default function BottomSheet({
   const isOrderAvailable = remainAmount <= 0;
```

Comment:
```suggestion
  const isOrderAvailable = orderType === 'TAKE_OUT' ? true : remainAmount <= 0;
```

제 생각에는 이렇게 하면 isDisabled 함수를 만들지 않고도 가능할 것 같아요
isDisabled에서 한 번 더 isOrderAvailable 값으로 검증하는 것이 아닌 주문 가능한지 여부를 판별할 때 한 번에 처리되는 것도 좋다고 생각되는데
성훈님 의견이 궁금합니다!

Reply:
오 이건 생각 못했네요 이걸로 바꾸겠습니다!